src/main/java/de/craftlancer/clfeatures/replicator/ReplicatorFeature.java

```diff
@@ -107,7 +115,9 @@ public CommandHandler getCommandHandler() {
     @Override
     public void remove(FeatureInstance instance) {
         if (instance instanceof ReplicatorFeatureInstance) {
-            ((ReplicatorFeatureInstance) instance).removeSpawnedItem();
+            ReplicatorDisplayItem displayItem = ((ReplicatorFeatureInstance) instance).getDisplayItem();
+            if (displayItem != null)
+                displayItem.remove();
```

Comment:
do this logic when destroying the feature instance, not when removing it from the manager